src/main/docker/Dockerfile

```diff
@@ -9,6 +9,9 @@ ADD run.sh /bin/run.sh
 ADD cpt4_5 /opt/athena/addons/cpt4_5
 ADD cpt4_4_5 /opt/athena/addons/cpt4_4_5
 
+ADD cpt4.jar /opt/athena/addons/cpt4_5
```

Comment:
FYI: Note that the Best practices for writing Dockerfiles suggests using COPY where the magic of ADD is not required